src/test/shell/bazel/bazel_test_test.sh

```diff
@@ -695,7 +695,7 @@ EOF
   assert_equals "fail" "$(awk "NR == $(wc -l < $TEST_log)" $TEST_log)"
 }
 
-function test_undeclared_outputs_are_zipped_and_manifest_exists() {
```

Comment:
Since we had this issue when developing, I think we can also add a test for the case that changing the value of `--zip_undeclared_test_outputs` will rerun the test.

Reply:
Done.